io.github.berarma.Oversteer.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: io.github.berarma.Oversteer
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: oversteer
+finish-args:
+  # Wayland support
+  - --socket=wayland
+  # X11 fallback support
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=/run/udev
+  # Access RPM leds
+  - --filesystem=/sys/class/leds
+  # Access the steering wheel parameters
+  - --filesystem=/sys/devices
+  - --device=all
+cleanup:
+  - '/include'
+  - '/lib/pkgconfig'
+  - '/man'
+  - '/share/cmake'
+  - '/share/doc'
+  - '/share/man'
+  - '/share/vala'
+  - '*.la'
+  - '*.a'
+modules:
+  - shared-modules/lzo/lzo.json
+  - name: cairo
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://cairographics.org/releases/cairo-1.18.0.tar.xz
+        sha256: 243a0736b978a33dee29f9cca7521733b78a65b5418206fef7bd1c3d4cf10b64
+  - python-dependencies.yaml
+  - name: oversteer
+    buildsystem: meson
+    config-opts:
+      - -Dudev_rules_dir=/etc/udev/rules.d
+    build-commands:
+      - ninja install
```

Comment:
this is done by "buildsystem: meson". It's not necessary.

Reply:
Removed.